core/resharding.go

```diff
@@ -164,9 +164,9 @@ func GetShardingStateFromBlockChain(bc *BlockChain, epoch *big.Int) (*ShardingSt
 	}
 	shardState = shardState.DeepCopy()
 
-	blockNumber := GetBlockNumberFromEpoch(epoch.Uint64())
-	rndSeedBytes := bc.GetVdfByNumber(blockNumber)
-	rndSeed := binary.BigEndian.Uint64(rndSeedBytes[:])
+	//blockNumber := GetBlockNumberFromEpoch(epoch.Uint64())
+	//rndSeedBytes := bc.GetVdfByNumber(blockNumber)
+	rndSeed := uint64(0)
```

Comment:
because randomness is not launched yet. Basically these code is wrong (GetBlockNumberFromEpoch is wrong). We better disable it than letting these code slip through launch. Added todo here. This is part of the randomness and resharding work.